web/impact/impact/tests/test_allocate_applications_view.py

```diff
@@ -0,0 +1,116 @@
+        judging_round.save()
+        judge_id = ExpertFactory().id
+        with self.login(email=self.basic_user().email):
+            url = reverse(AllocateApplicationsView.view_name,
+                          args=[judging_round.id, judge_id])
+            response = self.client.get(url)
+            assert response.status_code == 403
+            assert [JUDGING_ROUND_INACTIVE_ERROR.format(judging_round.id)
+                    in response.data]
+
+    def test_get_already_assigned(self):
+        context = JudgeFeedbackContext()
+        judging_round = context.judging_round
+        judge_id = ExpertFactory().id
+        with self.login(email=self.basic_user().email):
+            url = reverse(AllocateApplicationsView.view_name,
+                          args=[judging_round.id, judge_id])
+            response = self.client.get(url)
+            assert response.status_code == 200
+            count = len(response.data)
+            response = self.client.get(url)
+            assert response.status_code == 403
+            assert [ALREADY_ASSIGNED_ERROR.format(judge=context.judge.email,
+                                                  count=count)
+                    in response.data]
+
+    def test_get_no_app_for_judge(self):
+        context = JudgeFeedbackContext()
+        judge = ExpertFactory()
+        for app in context.applications:
+            JudgeApplicationFeedbackFactory(panel=context.panel,
+                                            judge=judge,
+                                            application=app)
+        with self.login(email=self.basic_user().email):
+            url = reverse(AllocateApplicationsView.view_name,
+                          args=[context.judging_round.id, judge.id])
+            response = self.client.get(url)
+            assert response.status_code == 403
+            assert [NO_APP_LEFT_FOR_JUDGE.format(context.judge.email)
+                    in response.data]
+
+    def test_get_adds_capacity_and_quota(self):
+        context = JudgeFeedbackContext()
+        judging_round = context.judging_round
+        judge = ExpertFactory()
+        commitment = JudgeRoundCommitmentFactory(judge=judge,
+                                                 judging_round=judging_round,
+                                                 capacity=0,
+                                                 current_quota=0)
+        with self.login(email=self.basic_user().email):
+            url = reverse(AllocateApplicationsView.view_name,
+                          args=[judging_round.id, judge.id])
+            response = self.client.get(url)
+            assert response.status_code == 200
+            commitment.refresh_from_db()
+            assert commitment.capacity > 0
+            assert commitment.current_quota > 0
+
+    def test_get_by_judge_succeeds(self):
+        context = JudgeFeedbackContext()
+        judging_round = context.judging_round
+        judge = ExpertFactory()
```

Comment:
Not a blocker, but would prefer to see a one-liner to get a loggable-in expert.

Reply:
Looked at this for a bit and I didn't see a quick way to do it.  I tried just adding `password="password"` which looks like it should work based on the code, but it doesn't and it smells like a rabbit hole.  Left a comment.